fixham/Makefile

```diff
@@ -0,0 +1,35 @@
+include ../metadata.mk
+
+NAME = fixham
+PACKAGE_NAME = github.com/projectcalico/calico/fixham
+
+include ../lib.Makefile
+
+.PHONY: build
+build: bin/hashrelease bin/release
+
+clean:
+	@rm -rf ./bin
+
+bin/release:
+	@mkdir -p bin
+	@go build -o bin/release ./release
+
+bin/hashrelease:
```

Comment:
In order to get Make to track rebuilds properly, you should have this depend on the files that you want to trigger rebuilds.

For example:

```
bin/hashrelease: $(shell find . -name "*.go")
```

^ This will rebuild bin/hashrelease whenever a .go file changes.